pkg/broker/binding_job.go

```diff
@@ -21,6 +21,7 @@
 package broker
 
 import (
+	"github.com/apex/log"
```

Comment:
Looks like github.com/apex snuck in there again

Reply:
Sigh I think it must be my goimports that auto runs. :(